roles/grafana/tasks/main.yml

```diff
@@ -102,7 +102,7 @@
     - plugins
 
 - name: grafana █ Ensure password is set to the one in the role variables
-  ansible.builtin.command: "grafana-cli admin reset-admin-password {{ grafana_security.admin_password }}"
+  ansible.builtin.command: "grafana-cli --homepath /usr/share/grafana admin reset-admin-password {{ grafana_security.admin_password }}"
```

Comment:
Shouldn't this --homepath be a role variable? I fear it is not the same on all OSes.
Thanks

Reply:
Note: no need to add vars for all OS versions, just add it for the OS you need, I will complete it once its merged 😉

Reply:
You're right.
@oxedions: shouldn't I provide a default location for all OSes to prevent failure on installations that didn't rely on this command to succeed?

Reply:
I've put a variable in defaults/main.yml. Is it ok?

Reply:
I would have liked them in:
https://github.com/bluebanquise/community/blob/main/roles/grafana/vars/[OS].yml
Could you do it? (sorry)

Reply:
No problem ;)